draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -491,22 +494,22 @@ congestion control to avoid overloading the network.
 
 QUIC is a congestion controlled transport protocol. Senders are required to
 employ some form of congestion control. The default congestion control specified
-for QUIC is an alogrithm similar to TCP NewReno, but senders are free to choose
+for QUIC is an alogrithm similar to TCP NewReno {{RFC9002}}, but senders are free to choose
 any congestion control algorithm as long as they follow the guidelines specified
 in {{Section 3 of ?RFC8085}}.
 
-RTP does not specify a congestion controller, but provides feedback formats for
-congestion control (e.g. {{!RFC8888}}) as well as different congestion control
+RTP does not specify a rate adaptation algorithm, but provides feedback formats for
+congestion control (e.g. {{!RFC8888}}) as well as different rate adaptation
```

Comment:
replace congestion control with rate adaptation?

Reply:
This is, of course, going to be maximally confusing to the reader, because RFC 8888 **says** it's enabling congestion control, but I think this is a good change to make for consistency in RTP-over-QUIC. 

Looking more closely at this text, RFC 8888 isn't technically part of "RTP", because it doesn't update any other RFC, so I think this is more accurate: 

> RTP itself does not specify a congestion control algorithm, but {{!RFC8888}} defines an RTCP
> feedback message intended to enable rate adaptation for interactive real-time traffic using RTP,
> and successful rate adaptation will accomoplish congestion control as well.
> Various rate adaptation algorithms for real-time media are defined in separate RFCs
> (e.g. SCReAM {{!RFC8298}} and NADA {{!RFC8698}}).

And, looking more closely at the Abstract and Introduction, I think I should be naming both congestion control and rate adaptation in both places, and then favoring the use of rate adaptation thereafter. 

Please let me know if this seems wrong or unhelpful.

Reply:
Sounds mostly fine to me, but I wonder if we need the part "and successful rate adaptation will accomoplish congestion control as well."? I think it might bring up the question of how exactly we define both and what the difference is.

Reply:
Oh, I just saw #61 is about this question. If we have some text early in the document that defines both terms, the question might have been answered already.